signac_dashboard/dashboard.py

```diff
@@ -382,9 +378,6 @@ def _job_search(self, query):
                     query = shlex.split(query)
                     f = signac.contrib.filterparse.parse_filter_arg(query)
                     flash(f"Search string interpreted as '{json.dumps(f)}'.")
-            if querytype == "document":
-                jobs = self.project.find_jobs(doc_filter=f)
-            else:
                 jobs = self.project.find_jobs(filter=f)
```

Comment:
Remove an indentation level.
```suggestion
            jobs = self.project.find_jobs(filter=f)
```